bankholidays/src/main/resources/bankholidays_classes.hbm.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0"?>
+<!DOCTYPE hibernate-mapping PUBLIC "-//Hibernate/Hibernate Mapping DTD//EN"
+        "http://hibernate.sourceforge.net/hibernate-mapping-3.0.dtd">
+<hibernate-mapping>
+  <class entity-name="Classes.BankHolidayClass" table="cel_bankholidays_bank_holiday">
+    <id name="id" type="integer" unsaved-value="undefined">
+      <column name="CEC_ID" not-null="true" />
```

Comment:
Why CEC? Usually we use something short to make field names recognisabe without knowing the table name. I would suggest BH (bank holiday) | CBH (celements bank holiday) | BHC (bank holiday class)